pymc4/random_variables/mixture.py

```diff
@@ -0,0 +1,77 @@
+from .random_variable import RandomVariable
+from tensorflow_probability import distributions as tfd
+import pymc4 as pm
+from .types import TensorLike
```

Comment:
Why does types need it's own module? In other words what does the layer of indirection add?

Reply:
scratch this, I see what you did.

Reply:
It's so I don't have to re-implement the same types in each module, mostly.

Reply:
But @twiecki's suggestion above is a good idea.